app/views/pseuds/_pseuds_form.html.erb

```diff
@@ -31,28 +32,34 @@
       <li><%= ts("Icons should be sized 100x100 pixels for best results") %></li> 
     </ul>
     <% if @pseud.icon_file_name %>
-      <%= f.check_box :delete_icon, {:checked => false} %>
-      <%= f.label :delete_icon, ts("Delete your icon and revert to our default") %></li>
```

Comment:
This </li> tag didn't belong here -- the list ended on line 33 and there was no opening <li> tag